qml/pages/MainPage.qml

```diff
@@ -60,7 +60,7 @@ Page {
             width: parent.width
 
             PageHeader {
-                title: "SailfishOS:Chum"
+                title: "SailfishOS:Chum GUI"
```

Comment:
I prefer it without GUI and having a shorter form: SailfishOS:Chum. But if you feel strongly about it, can be with GUI as well